.eslintrc

```diff
@@ -36,13 +36,6 @@
     }
   },
   "rules": {
-    "quotes": [

```

Comment:
This should be handled by prettier and prettier _sometimes_ wants double quotes so disabling here.